python/pyspark/sql/connect/readwriter.py

```diff
@@ -37,7 +37,7 @@
     from pyspark.sql.connect._typing import ColumnOrName, OptionalPrimitiveType
     from pyspark.sql.connect.session import SparkSession
 
-__all__ = ["DataFrameReader", "DataFrameWriter"]
+__all__ = ["DataFrameReader", "DataFrameWriter", "OptionUtils", "to_str"]
```

Comment:
Sorry if I missed sth but why do we need this? This isn't an API, and it's not being imported now.